qml/pages/ProgramsPage.qml

```diff
@@ -12,6 +12,7 @@ Page {
     property int offset: 0
     property int limit: 25
     property bool programsEnd: false
+    property string order: 'publication.starttime:desc'
```

Comment:
Could this be a property inside `yleApi`? Feels like a better place for it, or even all the others above could be moved there as "constants"? :thinking:

Reply:
This is constant only for now, later I still add settings for selecting sorting. Whether the order will then be stored in yleAPI or in a separate place needs to be decided.